src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@ CHANGELOG
 -----
 
  * added `AppVariable::getToken()`
- * Deprecated the possibility to inject the Form Twig Renderer into the form
-   extension. Inject it on TwigRendererEngine instead.
+ * [BC BREAK] Deprecated the possibility to inject the Form `TwigRenderer` into the form extension. 
```

Comment:
A deprecation is **not** a BC break. This is the whole point of deprecation: the old way still works.